cmd/edenPod.go

```diff
@@ -0,0 +1,546 @@
+package cmd
```

Comment:
Please split this file into smaller ones. Like "Datastore" "Network" "AppInstance" ...